### PR TITLE
style(Header): facelift, standardization and menu changes.

### PR DESCRIPTION
**Pull request by marcovincit**
This is a facelift PR, there are only visual design changes.

- User menu:
   - Goes to the left side with logo, standardizing dashboard and editor menu.
   - Add a menu link to go to website.
   - Add a menu link to go to new discover.
   
- Button component: 
   - Standardization of styles that are currently incorrect (sizes, border-radius).
   - Create ghost variant.
   
- Search:
   - Fixed alignment.
   - Visual changes.

# General:

### Current visual
current-header

### New visual
new-header

# User menu:

| Current      | New |
| ----------- | ----------- |
| current-header  | new-header   |

### CODE REVIEW

File: packages/app/src/app/components/dashboard/InputSelect.tsx [hunk 1]
```diff
@@ -15,7 +15,8 @@ const StyledSelect = styled.select`
   color: #e5e5e5;
   line-height: 24px;
   border: none;
-  border-radius: 2px;
+
```
Comment (olarclara) [inline packages/app/src/app/components/dashboard/InputSelect.tsx:18]:
```suggestion

```

**Review by olarclara — APPROVED** [review]

**Review by lenasmayor — APPROVED** [review]

File: packages/components/src/components/Button/index.tsx [hunk 2]
```diff
@@ -46,6 +46,19 @@ const variantStyles = {
...
+    color: 'mutedForeground',
+    // same technique as primary
+    ':hover:not(:disabled)': {
+      color: 'foreground',
+      backgroundColor: '#E5E5E51A',
+    },
+    ':focus:not(:disabled)': {
+      color: 'foreground',
+      backgroundColor: '#E5E5E51A',
+    },
```
Comment (olarclara) [inline packages/components/src/components/Button/index.tsx:60]: Does the following work?

```suggestion
    ':hover:not(:disabled), :focus:not(:disabled)'': {
      color: 'foreground',
      backgroundColor: '#E5E5E51A',
    },
```

File: packages/components/src/components/Button/index.tsx [hunk 3]
```diff
@@ -88,18 +101,17 @@ const commonStyles = {
...
   fontWeight: 'medium',
-  lineHeight: 1, // trust the height
+  lineHeight: '16px', // trust the height
   border: 'none',
-  borderRadius: 'small',
-  transition: 'all ease-in',
+  borderRadius: '4px',
+  transition: 'background .3s, color .3s',
   textDecoration: 'none',
   transitionDuration: theme => theme.speeds[2],
```
Comment (olarclara) [inline packages/components/src/components/Button/index.tsx:116]: Does the duration (`300ms`) defined in line 114 gets overridden by the transition duration (`theme.speeds[2] = 100ms`) set in line 116?

File: packages/app/src/app/components/Notifications/index.tsx [hunk 4]
```diff
@@ -7,7 +7,7 @@ import css from '@styled-system/css';
 import { Overlay } from 'app/components/Overlay';
 import { NotificationsContent } from './Content';
 
-export const Notifications = () => {
+export const Notifications = ({ dashboard }: any) => {
```
Comment (olarclara) [inline packages/app/src/app/components/Notifications/index.tsx:10]:
```suggestion
export const Notifications = ({ dashboard }: { dashboard?: boolean }) => {
```

**Review by olarclara — CHANGES_REQUESTED** [review]
Can you repro this issue? Seems that only the create button and the search render at first causing the screen to glitch.

https://user-images.githubusercontent.com/24959348/213503828-4b02be2d-b03e-414a-a6e3-e6b285405041.mov

**Review by olarclara — APPROVED** [review]
Looks great!

**Pull request merged**